src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssStyleObject = {};
+  const declarations = sourceString.split(';');
+
+  declarations.forEach((declaration) => {
+    const newDeclaration = declaration.trim();
```

Comment:
Here's an example of how to filter out empty lines to make your code more robust.

```suggestion
  declarations
  .map((declaration) => declaration.trim())
  .filter((declaration) => declaration)
  .forEach((declaration) => {
    const newDeclaration = declaration.trim();
```